domain/src/main/java/com/lcw/study/nextstep/domain/Calculator.kt

```diff
@@ -0,0 +1,69 @@
+package com.lcw.study.nextstep.domain
+
+class Calculator {
+    private var value = 0
+    private var operation: OperationType =OperationType.PLUS
+    private var number: Int = 0
+
+    fun evaluate(inputText: String?): Int {
+
+        divideTextType(inputText)
+
+        return value
+    }
+
+    private fun divideTextType(inputText:String?){
+        if (inputText.isNullOrEmpty()) {
+            throw IllegalArgumentException("입력값이 null이거나 빈 공백 문자 입니다.")
+        }
+        val splitTexts: List<String> = (inputText.split(" "))
```

Comment:
```suggestion
        val splitTexts: List<String> = inputText.split(" ")
```
불필요하게 괄호가 들어가있는것 같아요.

Reply:
ce4b2e5
수정하였습니다!